catalog/USGS/USGS_NLCD_RELEASES_2019_REL_NALCMS.jsonnet

```diff
@@ -0,0 +1,269 @@
+    international organization created by the Canada, Mexico, and United States governments 
+    under the North American Agreement on Environmental Cooperation to promote environmental 
+    collaboration between the three countries.
+
+    The general objective of NALCMS is to devise, through collective effort, a harmonized 
+    multi-scale land cover monitoring approach which ensures high accuracy and consistency 
+    in monitoring land cover changes at the North American scale and which meets each country’s 
+    specific requirements.  
+
+    This 30-meter dataset of North American Land Cover reflects land cover information for 2020 
+    from Mexico and Canada, 2019 over the conterminous United States and 2021 over Alaska. 
+    Each country developed its own classification method to identify Land Cover classes and 
+    then provided an input layer to produce a continental Land Cover map across North America. 
+    Canada, Mexico, and the United States developed their own 30-meter land cover products; 
+    see specific sections on data generation below.
+
+    The main inputs for image classification were 30-meter Landsat 8 Collection 2 Level 1 data in the 
+    three countries (Canada, the United States and Mexico). Image selection processes and reduction to 
+    specific spectral bands varied among the countries due to study-site-specific requirements. 
+    While Canada selected most images from the year 2020 with a few from 2019 and 2021, the Conterminous 
+    United States employed mainly images from 2019, while Alaska land cover maps are mainly based on the 
+    use of images from 2021. The land cover map for Mexico was based on land cover change detection 
+    between 2015 and 2020 Mexico Landsat 8 mosaics.
+
+    In order to generate a seamless and consistent land cover map of North America, national maps were 
+    generated for Canada by the CCRS; for Mexico by CONABIO, INEGI, and CONAFOR; and for the United States 
+    by the USGS. Each country chose their own approaches, ancillary data, and land cover mapping methodologies
+    to create national datasets. This North America dataset was produced by combining the national land cover datasets. 
+    The integration of the three national products merged four Land Cover map sections, Alaska, Canada, 
+    the conterminous United States and Mexico.
+
+    See also:
+
+    * Natural Resources Canada has North American Land Cover information available online at 
+      (https://open.canada.ca/data/en/dataset/ee1580ab-a23d-4f86-a09b-79763677eb47)
+ 
+    * The National Commission for the Knowledge and Use of Biodiversity has North American Land Cover 
+    information available online at (https://www.biodiversidad.gob.mx/monitoreo/cobertura-suelo)
+
+    * The U.S. Geological Survey has North American Land Cover information available online at (www.mrlc.gov)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
```

Comment:
Sorted order and comment out proposed new keywords. We can discuss new keywords after the dataset is in.  The current keyword list is here: https://developers.google.com/earth-engine/datasets/tags
```
  keywords: [
    // 'land_cover',
    // 'land_surface',
    // 'land_use',
    'landsat',
    // 'north_america',
    'reflectance',
    // 'remote_sensing',
  ],
```

Reply:
That sounds good.
I updated it to this for now:

 keywords: [
  'landsat',
  'reflectance',
  'landcover',
  'nlcd',
   // 'land_surface',
   // 'land_use',
   // 'north_america',
   // 'remote_sensing',

Reply:
Try to keep them sorted.  e.g. like this:

https://github.com/google/earthengine-catalog/blob/6f19d2dcf50c8da18204086f6a885b83d068c8ff/catalog/USGS/USGS_NLCD_RELEASES_2019_REL_NALCMS.jsonnet#L97

```
  keywords: [
    // 'land_cover',
    // 'land_surface',
    // 'land_use',
    'landcover',
    'landsat',
    'nlcd',
    // 'north_america',
    'reflectance',
    // 'remote_sensing',
  ],
```

Reply:
Updated